plugin/core/views.py

```diff
@@ -997,6 +998,34 @@ def is_location_href(href: str) -> bool:
     return href.startswith("location:")
 
 
+def starts_with_custom_uri_scheme(href: str) -> bool:
+    return urlparse(href).scheme.lower() not in ("", "http", "https")
+
+
+def row_col_from_uri_fragment(href: str) -> Tuple[Optional[int], Optional[int]]:
```

Comment:
I've only skimmed over the code for now, but iirc I had already implemented the row/col parsing in https://github.com/sublimelsp/LSP/blob/c06d811dc0bd9f1176f349ac6f82ae84dba94a67/plugin/core/open.py#L28-L47
which should cover all combinations that are supported by VS Code. Could that be reused after moving this local function to somewhere else to make it available from the outside?

Reply:
Nice, I missed that functionality. I’ll reuse that.